### ISSUE TITLE
Netherlands: GoodFriday should by of type PublicHolidayType.School

### ISSUE THREAD

**Issue by marcelerkel** [nager/Nager.Date#498]
### Affected country

Netherlands

### Affected public holiday

GoodFriday

### Source of the information

https://en.wikipedia.org/wiki/Public_holidays_in_the_Netherlands

### More information

In the Netherlands GoodFriday is a school holiday, not a public holiday.

From the wikipedia source:

| Date | English name | Dutch name | Notes |
|-----|------|-----|-----|
| March or April | Good Friday | Goede Vrijdag | This is a national holiday, but not a mandatory paid holiday and a normal workday for most employees.[1] It is a popular day off due to the Easter weekend, especially the combination with Easter Monday. |

https://github.com/nager/Nager.Date/blob/main/src/Nager.Date/PublicHolidays/NetherlandsProvider.cs

**Comment by tinohager** [comment]
I have checked our old issues and we have changed them 2021, what is right now I don't want to configure it back and forth all the time if possible.

https://github.com/nager/Nager.Date/issues/314

@dennisseders Would you like to discuss and help find a solution here?

**Comment by marcelerkel** [comment]
I agree with Dennis that the information provided by the Dutch government should be the source of truth and not wikipedia. I also agree that according to the Dutch government Good Friday is a public holiday. However, according to the following document [Is Goede Vrijdag een vrije dag?](https://www.rijksoverheid.nl/onderwerpen/arbeidsovereenkomst-en-cao/vraag-en-antwoord/goede-vrijdag-vrije-dag) (Is Good Friday a day off?) it is not a day off for employees of either companies or the government.

My understanding is that `PublicHolidayType.Public` is used when the majority of people have a day off from work.

Since English is not my native language (I'm Dutch) I googled the word Observance and found the following link which seems to deal with the same stuff as Nager.Date: [What is the difference between an Observed Holiday and an Oservance](https://www.predicthq.com/support/what-is-the-difference-between-an-observed-holiday-and-an-observance).
This link states the following:

_Observances, on the other hand, are remembrance, awareness and celebration days or events that are not normally days off work, like Valentine's Day, Mother's Day or Father's Day._

I think this is similar to how Good Friday is celebrated in the Netherlands, normally not a day off work except for schools.

So if my understanding about `PublicHolidayType.Public` and `PublicHolidayType.Observance` is correct, then Good Friday falls in the categories `PublicHolidayType.Observance` and `PublicHolidayType.School`.

I don't use Nager.Date myself. The public api was mentioned in a topic on the openHAB forum and I got curious so I checked the result for the Netherlands and thought and think that it is currently incorrect. In its current state I wouldn't use it to disable my alarm clock on public holidays because on Good Friday I'd be late at work :)

@dennisseders Since I don't use Nager.Date, what do you think? Change the type of Good Friday to Observance | School or keep it as it is?

**Comment by tinohager** [comment]
openHAB backlink, https://community.openhab.org/t/public-holiday-api-and-http-binding/146533/4

**Comment by tinohager** [comment]
In my point of view, a `public holiday -> PublicHolidayType.Public` should be free for all.

**Comment by tinohager** [comment]
In the list from the European Commission the `Good Friday` is also not listed.

https://ec.europa.eu/taxation_customs/dds2/rd/publicholidays_consultation.jsp?Lang=en&Expand=true&DesLang=EN&Country=NL

**Comment by dennisseders** [comment]
The Dutch government [states](https://www.government.nl/topics/working-hours/question-and-answer/public-holidays-in-the-netherlands) that Good Friday is a public holiday. Since [#315 ](https://github.com/nager/Nager.Date/pull/315) was merged Nager.Date follows this.

The Dutch government also [states](https://www.government.nl/topics/working-hours/question-and-answer/public-holidays-in-the-netherlands) there is no law that says employees must be given a day off work on certain public holidays. So no one has a legal right to a day off on a public holiday. 

The Dutch government also [states](https://www.rijksoverheid.nl/onderwerpen/arbeidsovereenkomst-en-cao/vraag-en-antwoord/goede-vrijdag-vrije-dag) (a default English translation is not available) that schools are almost _always_ closed on Good Friday but that the particular school should be contacted to be sure about this.

To summarize:

1. Nager.Date considers Good Friday as a public holiday just like the Dutch government does
2. A public holiday is not a day off for everyone in the Netherlands
3. Almost all schools are closed on Good Friday

Haven this said:

1. Do not change how Nager.Date considers Good Friday as a holiday of `PublicHolidayType.Public` in the Netherlands as it follows the Dutch government's definition.
2. Do not change how Nager.Date considers Good Friday as a holiday of `PublicHolidayType.School` as it can be that not all school follow this.

**Comment by tinohager** [comment]
@dennisseders thank you for your detailed answer. Then I will close the issue with it and we leave everything as it is.
